### PR TITLE
Trunk 5959 removed unused imports

### PR DESCRIPTION
**Pull request by insookwa**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5959

## Checklist: I completed these to help reviewers :)

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		(filter*),
 		(filter-mapping*),
 		(messages*),
-		(mappingFiles?)
+		(mappingFiles?),
```
Comment (sacull) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.5.dtd:30]: Why did you change this file?

File: api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd [hunk 2]
```diff
@@ -26,8 +26,8 @@
 		(filter*),
 		(filter-mapping*),
 		(messages*),
-		(mappingFiles?)
-		(packagesWithMappedClasses?)
+		(mappingFiles?),
+		(packagesWithMappedClasses?),
```
Comment (sacull) [inline api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd:30]: And the file too?

**Comment by sacull** [comment]
You have indicated that your branch is based on the current master version, but from what I can see it is based on a month ago version. Are you sure you made `git pull --rebase upstream` master as indicated in the description?

File: api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd (hunk 2 above)
Comment (insookwa) [reply api/src/main/resources/org/openmrs/module/dtd/config-1.6.dtd:30]: they were showing an error and thought they were the reason my build was failing, can I make the change and make the pull request again

**Comment by insookwa** [comment]
> You have indicated that your branch is based on the current master version, but from what I can see it is based on a month ago version. Are you sure you made `git pull --rebase upstream master` as indicated in the description?
yes, I did but I think it checked with the repo i folked to my account because I did that a month ago. maybe to ask does the repository i folked update automatically when changes are made on the master repository 
what can I do to fix this

**Comment by insookwa** [comment]
> > You have indicated that your branch is based on the current master version, but from what I can see it is based on a month ago version. Are you sure you made `git pull --rebase upstream master` as indicated in the description?
> > yes, I did but I think it checked with the repo i folked to my account because I did that a month ago. maybe to ask does the repository i folked update automatically when changes are made on the master repository
> > what can I do to fix this

C:\Users\Insookwa\openmrs-core>git pull --rebase upstream master
From https://github.com/insookwa/openmrs-core
 * branch                  master     -> FETCH_HEAD
Already up to date.

**Comment by sacull** [comment]
Show result: `git remote -v`

**Comment by insookwa** [comment]
> Show result: `git remote -v`

```
C:\Users\Insookwa\openmrs-core>git remote -v
origin  https://github.com/insookwa/openmrs-core.git (fetch)
origin  https://github.com/insookwa/openmrs-core.git (push)
upstream        https://github.com/insookwa/openmrs-core.git (fetch)
upstream        https://github.com/insookwa/openmrs-core.git (push)
```

**Comment by sacull** [comment]
Do:
`git remote remove upstream`
`git remote add upstream https://github.com/openmrs/openmrs-core.git`

Only then was the command `git pull --rebase upstream master` will download the latest version of the master.

You can find more here: https://wiki.openmrs.org/display/docs/Using+Git

**Comment by insookwa** [comment]
Thanks @sacull  I've done that. so should I close this pull request and work on a new one with updated data

**Comment by sacull** [comment]
You can work on the same after the update, and you can close this PR and make a new one. It depends only on you. ;)

**Comment by insookwa** [comment]
being my first time to do. this There are a number of things I had not considered on making the pull request. Thanks for the advice. I am closing this to make a better pull request. thanks @sacull
